torch/fx/passes/utils/matcher_utils.py

```diff
@@ -91,6 +92,18 @@ def __init__(self, pattern: Graph,
             # and should be matched against as an anchor
             self.pattern_anchors = [n for n in output_node.all_input_nodes if len(n.users) == 1]
 
+    def _match_attributes(self, pn: Node, gn: Node) -> bool:
+        # Attributes matching is compilcated. Right now we only support matching constant tensor
+        pn_value = getattr(pn.graph.owning_module, pn.target)
+        gn_value = getattr(gn.graph.owning_module, gn.target)
+        if type(pn_value) != type(gn_value):
+            return False
+        if isinstance(pn_value, torch.Tensor):
+            return isinstance (gn_value, torch.Tensor)
+        else:
```

Comment:
We should be able handle python literals, int/float/bool ect...

Reply:
Make sense. For python literals, i'm thinking they should be pass if the type is the same maybe?